content/c-sharp/concepts/arrays/terms/resize/resize.md

```diff
@@ -0,0 +1,101 @@
+---
+Title: '.Resize()'
+Description: 'Updates or defines the size of an existing array.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Resize()`** method changes the number of elements of a one-dimensional array to the specified new size.
+
+## Syntax
+
+```pseudo
+// Sets a new size for the specified array
+Array.Resize<int>(ref int[] array, int newSize);
+```
+
+`.Resize()` is a modifier of the `Array` object.  When used as part of a one-dimensional array, it defines the number of elements acceptable by the array, or redefines the size of an array.  This can be used to both lengthen or shorten the number of elements within a given array.
+
+## Example
+
+The following example :
+
+```cs
+using System;
+
+public class Example
+{
+   public static void Main()
+   {
+        //Create and initialize an example string array.
+        String[] planets = {"Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus", "Neptune", "Pluto"};
+
+        //Display the values of the array.
+        PrintIndexAndValues(planets);
+
+        //Resize the array to a bigger size (adding three elements).
+        Array.Resize(ref planets, planets.Length + 3);
+
+        //Display the values of the new (larger) array.
+        PrintIndexAndValues(planets);
+
+        //Resize the array to a smaller size (eight total elements).
+        Array.Resize(ref planets, 8);
+
+        //Display the values of the new (smaller) array.
+        PrintIndexAndValues(planets);
+   }
+   
+    public static void PrintIndexAndValues(String[] planets)  {
+        for(int i = 0; i < planets.Length; i++)
+        {
+            Console.WriteLine("   [{0}] : {1}", i, planets[i]);
+        }
+        Console.WriteLine();
+    }
+}
+/*
```

Comment:
Instead of a comment, we like the output of the example to be in it's own separate ```shell block with an introductory sentence such as "This example results in the following output:"

Reply:
The output of the example has been placed in pseudocode shell blocking instead of how it was written previously, and updated with the output of the new example, along with the introductory sentence.